build.sbt

```diff
@@ -23,18 +23,26 @@ lazy val rocketchip = RootProject(file("rocket-chip"))
 lazy val testchipip = project.settings(commonSettings)
   .dependsOn(rocketchip)
 
-lazy val example = (project in file("."))
+def dependsOnRocket(prj: Project): Project = {
```

Comment:
Probably just a nit, but it might be good to either change the name of this function, or add a comment that makes it explicit that this will ignore the submodule-based projects defined above when if using published versions.

Reply:
Good point, hopefully what I've added is better. Also, it technically depends on testchipip so saying `dependsOnRocket` is a bit confusing.